docs/hw/hookup.md

```diff
@@ -1,87 +1,21 @@
-# iRobot® Create® 3 Hookup Guide
+# iRobot® Create® 3 Hardware Hookup Overview
+The Create® 3 is based on the Roomba®, a robot vacuum cleaner.
+Its sensors, actuators, and compact design are capable of navigating and mapping a the whole floor of a home or office space.
+The robot also ships with an iRobot® Home Base™ Charging Station.
 
```

Comment:
Add a another sentence or two here, something like "It is possible to directly connect various single board computers to the Create 3 robot in order to provide on-board intelligence. Here are a few examples."

Reply:
Added.